img/e/ets.org.svg

```diff
@@ -0,0 +1 @@
13.1 1.1-5.2 2.1-10.9 2.4-12.6.3-1.7 1.3-6.6 2.1-11 .9-4.3 1.8-9.5 2.1-11.4.2-1.9.7-4.2.9-5 .2-.8.7-2.9 1-4.5.2-1.7.7-4.1 1-5.5.3-1.4.8-4.1 1.1-6 .2-1.9.7-4.2.9-5 .2-.8.7-3.1 1-5 .3-1.9 1.8-9.4 3.1-16.5 1.4-7.2 2.8-14.1 3-15.5.2-1.4.6-3.6.9-5 .3-1.4 1.4-7.2 2.5-13s2.2-11.6 2.5-13c.3-1.4.7-3.9 1-5.5.4-2.9.8-4.8 2-10.5.8-4 1.4-7.2 2-11 .6-4 .9-5.6 2.3-12 .6-2.5 1.3-6.1 1.6-8 .3-1.9.7-4.4 1-5.5.2-1.1.7-3.9 1.1-6.3l.7-4.2H580l.6-2.8c.4-1.5 1.7-5.1 3-8 1.3-2.9 2.3-5.5 2.3-5.9-.1-.3.3-.9.8-1.2.5-.3 2-2.2 3.3-4.1 3.2-4.7 11.7-13 13.1-13 .7 0 .9-.3.6-.7-1.3-1.2 11.8-8.8 21.8-12.7 2.5-.9 3.3-1.5 2-1.6-5.4-.3-215.4-.5-216.7-.2zM222.9 353.2c0 .2-.2 1.4-.4 2.8s-1.3 7.2-2.4 13c-1.2 5.8-2.4 11.8-2.7 13.5-.2 1.6-.7 3.7-.9 4.5-.2.8-.7 3.7-1.1 6.5-.3 2.7-.8 5.3-1 5.6-.2.3-.6 2.1-.9 4-.3 1.9-1.3 7.2-2.1 11.9-.9 4.7-1.8 9.6-2 11-.3 1.4-.7 3.8-1.1 5.5-.9 4-.8 3.4-1.8 9-.4 2.7-.9 5.4-1.1 6-.1.5-.5 3-.9 5.5-.4 2.4-.8 4.7-1.1 5.1-.2.3-.5 2.2-.8 4.2-.2 2-.7 4.4-1 5.4-.2 1-.7 3.4-1 5.3-.5 3.3-.9 5-2.1 11-.3 1.4-.8 4.3-1.1 6.5-.3 2.2-.7 4.2-.9 4.5-.2.3-.6 2.6-1 5-.3 2.5-1.1 6.5-1.7 9-.5 2.5-1.2 5.8-1.4 7.5-.3 1.6-1.4 7.5-2.4 13-2.3 11.6-3.1 16.2-3.5 19-.1 1.1-.3 2-.5 2-.1 0-.3.9-.4 2-.2 1.1-.6 3.3-.9 5-.4 1.6-1 4.7-1.3 6.9-.3 2.1-.8 4.8-1 6-1.2 5.4-2.5 12.4-3 15.6-.3 1.9-1.7 9.1-3 15.8-1.3 6.8-2.4 12.6-2.4 12.8 0 .2 39.4.4 87.5.4 69.1 0 87.7-.3 88-1.3.3-.6.7-2.8.9-4.7.3-1.9.5-3.5.6-3.5.1 0 .3-1.1.6-2.5.2-1.4.9-5 1.5-8s1.2-6.6 1.5-8c.2-1.4.9-5 1.5-8s1.2-6.6 1.5-8c.2-1.4.7-3.1.9-3.8.5-1.4 5.8-1.3-65.7-1.3H242l.5-3.7c.3-2 .7-4.8 1-6.2.7-3.5 5.5-29 6-32 .3-1.4.7-3.4.9-4.5.2-1.1.7-4.4 1.1-7.3.4-3 1.3-5.6 2-5.8.6-.3 24.3-.6 52.5-.7l51.4-.2.8-3c.4-1.7 1-4.6 1.3-6.5.5-4.1 1.3-8.2 2.2-12.5.4-1.7.8-3.6.9-4.3.1-.8.8-4.6 1.6-8.5.7-4 1.2-7.8 1.1-8.5 0-.6.2-1.2.6-1.2.4 0 .7-1 .6-2.2l-.2-2.2-51.4-.1c-28.3-.1-51.7-.3-52-.3-.4-.1 0-3.2.7-6.8.8-3.6 1.6-8 1.8-9.7.3-1.8 1-5.7 1.6-8.7.6-3 1.4-6.9 1.7-8.5.3-1.7 1.2-6.2 1.9-10 .8-3.9 1.6-7.5 1.6-8.1.1-.9 14.1-1.2 51.7-1.2 28.4 0 53-.1 54.7-.1 3-.1 3.1-.2 4.4-7.4.8-3.9 1.5-7.7 1.6-8.2.1-.6.5-2.7.8-4.9.3-2.1.8-4.1 1-4.5.2-.3.7-2.8 1-5.5.4-2.7.9-5.2 1.1-5.5.1-.3.6-2.4.9-4.5.4-2.1.9-5 1.2-6.2l.6-2.4h-84.1c-46.3 0-84.1.1-84.2.2zM896 616.5c-21.1 4.4-33.6 23.9-28.5 44.5 2.6 10.5 12 20.4 22.9 24 9 2.9 24.5 1.1 29.7-3.6.8-.8 1.8-1.4 2.2-1.4 1.3 0 7.7-7.1 10.1-11.2 4.9-8.6 6-19 3.1-29.3-2.7-9.7-13.6-20.1-23-22-2.2-.5-4.7-1-5.5-1.2-3.7-.9-5.9-.9-11 .2zm11 4c10.8 1.9 21.5 12.6 23.6 23.5 2 11-1.3 21.7-8.8 28.4-8.4 7.6-18.5 10.2-29.2 7.6-13-3.1-23.4-17-22.8-30.5.2-5.8 3-14.5 4.6-14.5.5 0 .7-.3.3-.6-.8-.8 7.3-9.4 8.9-9.4.7 0 1.4-.4 1.6-.9.2-.5 3-1.8 6.3-2.9 5.5-1.9 8.1-2 15.5-.7z"/><path d="M885.9 631.9c-.4 6.8-.3 37.1.1 37.2.3 0 1.3.2 2.4.4 3.8.7 4.4-.4 4.3-7.8l-.2-7.2h6.3c8.1 0 10.4 2 10.6 9.6.2 5.3.3 5.4 3.1 5.4 2.6.1 3.1-.4 3.5-3 .9-5-1.1-11.3-4.4-13.4l-2.9-1.9 3.1-1.6c3.7-1.9 4.4-2.9 4.9-7.3.9-8.6-4.8-12-20.4-12-8.8 0-10.3.2-10.4 1.6zm22.2 5.6c2.4 2.3 2.2 6.9-.6 9.5-1.9 1.7-7.5 2.6-13.2 2.1-.9-.1-1.3-2.3-1.3-7.2v-7.2l6.5.6c4.3.3 7.3 1.1 8.6 2.2z"/></svg>
```

Comment:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 959.1 662.9"><path d="M664.4 0c0 .1-8.6.6-18.8.9a840 840 0 0 0-22 1c-1.9.3-6.6.7-10.5 1l-9 1-7 1a352 352 0 0 0-17 2.6c-2.7.5-6.7 1.2-8.7 1.4l-8.5 1.7-7.8 1.6c-4.2.7-27.5 6.7-36 9.2l-8.9 2.6c-9.9 2.3-46.2 16-65.6 24.7-7.2 3.3-27 13.2-33.5 16.8l-9.5 5.2c-6 3.2-33.3 20.5-37 23.5l-11.7 8.7-12.5 9.5-4.8 4.1a270.5 270.5 0 0 0-22.4 20.6c-19 18.9-20 20.2-18.7 23l1.1 2.6 3-2.4c1.5-1.3 5.1-3.9 8-5.8a22 22 0 0 0 5-4.3c0-.4.6-.7 1.4-.7.8 0 1.6-.4 1.8-.9.1-.5 6.2-4.7 13.3-9.4a440.6 440.6 0 0 0 16-10.7c.2 0 3-1.6 6-3.5s6-3.5 6.5-3.5 1-.4 1-.8 1.5-1.5 3.3-2.3a225.1 225.1 0 0 0 23.7-12.9c.5 0 5.5-2.2 11-4.9 5.5-2.8 10.3-5 10.5-5.1.3 0 2.8-1.1 5.7-2.4l14.8-6.1 12.8-5.2c1.8-.8 4-1.6 5-1.8 1-.2 7.1-2.2 13.7-4.5a577 577 0 0 1 33.5-10l6.5-1.7 3-.7a524 524 0 0 1 16-3.6 349.3 349.3 0 0 0 14-2.5 159.2 159.2 0 0 1 18.1-2.9 380 380 0 0 1 34.6-4.1c14.8-1.3 20.5-1.5 40.3-1.5 20.2 0 26 .2 39.5 1.5a215.8 215.8 0 0 1 24 2.9l6.6 1 5.5 1c12.6 2.1 36.1 8.4 48 12.8 38.7 14.4 65.2 30.8 89.3 55.2 14.2 14.6 14.6 15 14.6 16.4 0 .7.4 1.2 1 1.2 1 0 10 13 14.7 22a198.7 198.7 0 0 1 21.8 74l1.1 11.5c.7 5.7.6 32.7 0 41a355.5 355.5 0 0 1-3 27l-1.2 7.5-1.3 6.5c-.8 4.5-2.4 11-3.6 15.8l-1 3.8c-1 4.5-3.8 13.8-7.5 24.5a292 292 0 0 1-45 83.8c-2.3 3-2.7 4.2-1.9 5.7.5 1.1 1.4 1.9 2 1.9 1.8 0 37.8-36.7 50.6-51.6l6.8-7.9c25.2-29.1 49.3-68 60.4-97.2a262.4 262.4 0 0 0 11.5-42.3l1.5-10.5c1-6.3.7-36.8-.3-44.4a223.3 223.3 0 0 0-11-45.6 242.8 242.8 0 0 0-33.5-62.5c-6.9-9-27.9-30.6-34.5-35.6a94.7 94.7 0 0 1-5.6-4.4c-.8-.8-3.6-2.8-6-4.5a57 57 0 0 1-5-3.5 31 31 0 0 0-8-5 1 1 0 0 1-1-1c0-.6-.4-1-.9-1-.6 0-2.9-1.3-5.3-2.8-2.4-1.5-8.3-4.8-13.3-7.2a56.7 56.7 0 0 1-9.3-5.3c-.2-.4-1-.7-2-.7a5 5 0 0 1-2.7-.8 26 26 0 0 0-5.5-2.6l-7.5-3.1a310.6 310.6 0 0 0-44.6-14.1c-.5-.3-2.2-.8-3.7-1l-6.2-1.4c-1.9-.5-5-1.1-7-1.4l-6.5-1.2c-3.7-.7-5.7-1-12-1.8l-7.5-1.2c-1.4-.3-5.4-.7-9-1a347.8 347.8 0 0 0-27.5-2 267.9 267.9 0 0 1-18.7-.9z"/><path d="M303.1 92.1a816.8 816.8 0 0 0-86 43.1l-3.5 1.9c-.8.4-1.7 1-2 1.3a714.7 714.7 0 0 0-72.5 50.5l-6.8 5.6-4 3.5-9.8 8.5c-10.5 9.3-31.2 30-39.9 40-4 4.7-7.7 8.6-8.2 8.8-.4.2-.8.8-.8 1.3 0 .6-.9 2-2 3.2-8 8.5-34 46.4-34 49.5 0 .6-.4 1.2-.8 1.4-2 .8-20.7 41.7-21.6 47.3a55.8 55.8 0 0 0-3 9.6L5.4 378 4 383.5a162 162 0 0 0-3.2 16c-1.3 5-1.2 51.2.2 57.8a230 230 0 0 1 2 10.3c.6 4.7 3 14.1 5.7 22.9a250 250 0 0 0 10.7 27A208.4 208.4 0 0 0 85 595.1c1.3 1.5 17.6 12.3 24 16 2.8 1.5 5.2 3.2 5.3 3.6.2.4 1.2.8 2.2.8 1 0 2 .3 2.2.7.5 1.2 18 9.6 30.4 14.7a547 547 0 0 0 36.4 13c5 1.7 12 3.4 12 2.9 0-.3.7 0 1.5.7s1.5 1 1.5.7c0-.3 1.5-.1 3.3.4 1.7.6 4.1 1.2 5.2 1.4l4.3 1a287.4 287.4 0 0 0 24 4.5c.3.4 1.3.5 7.2 1.4l7.5 1.1a523.6 523.6 0 0 0 79 4.9 830.5 830.5 0 0 0 56.4-1.9c3.5-.2 8-.7 10-1l8.6-1.1a293 293 0 0 0 17-1.9c5.3-.9 8-1.2 14-2 2.8-.3 5.7-.8 6.5-1l5.5-.9c2.2-.4 6.3-1 9-1.6l8-1.4c1.7-.2 3.7-.7 4.5-1a37 37 0 0 1 5.5-1.1c2.2-.3 5.8-1 8-1.6a88.9 88.9 0 0 1 11-2.3l5.3-1.1 4.2-1 4.3-1 5-1.1c1.4-.3 2.8-.4 3-.3.1.1 1.1-.3 2.3-.9 1.1-.6 3.2-1.3 4.5-1.6 4.6-1 6.5-1.4 16-4.1a794.5 794.5 0 0 0 62.8-21.6l15.6-6.3c8.2-2.6 50.2-23.4 68.6-34l17.8-10.2 4.5-2.6 18.1-12.1a629.4 629.4 0 0 0 62.5-46.7l6.6-5.5 9-8c10-8.9 11.5-10.7 10.2-12.2-.6-.7-1.5-1.3-2-1.3-.8 0-9.3 6.2-16.8 12.2-3.7 2.9-45.6 30.8-46.4 30.8-.4 0-2.3 1.1-4.4 2.4-5 3.3-11.9 7.2-16.9 9.7-2.4 1.2-4.3 2.5-4.3 3s-.6.9-1.4.9c-.7 0-4.4 1.7-8.2 3.8a720.2 720.2 0 0 1-79.4 35.7 803.8 803.8 0 0 1-58.5 19.1l-7.5 1.9-6 1.4-6.3 1.7-3.5.8-4.2 1-4.7 1-4.5 1.1-7.8 1.7c-3 .6-5.8 1.1-6.2 1a9 9 0 0 0-3.2.7c-1.3.6-4 1.2-6.3 1.6l-6.3 1a732.1 732.1 0 0 1-52.5 7.9c-25.6 3-44.6 4.1-76 4-23.6 0-46-.8-52.5-1.8-1.4-.3-5.5-.7-9.2-1.1-3.7-.3-8-.8-9.5-1.1l-6.3-.9c-5.5-.5-12-1.5-18-2.7l-7.5-1.4-9.5-1.9-5-1.1-4.2-1-3.5-.9c-2.7-.5-11.9-3.1-19.8-5.5l-10.5-3.2c-1.9-.5-3.9-1.3-4.5-1.6a318 318 0 0 1-39-16.7c-64.5-32.6-100.6-79.1-110-142a206.8 206.8 0 0 1 1.5-62.3l1.6-8.7a259.7 259.7 0 0 1 13.4-44.5c5-12.9 19.3-42 21.3-42.8a347.1 347.1 0 0 1 23.3-35.2 565.2 565.2 0 0 1 71.7-74.2c1-1 2.1-1.8 2.5-1.8.4 0 2.3-1.5 4.2-3.3 2-1.8 6.2-5.3 9.5-7.8l8-6.1c2.6-2.2 6-4.5 20.3-14.3l12.2-8.5c.3-.3 4.8-3.2 10-6.4l13-8.1c6-3.8 18.3-10.9 23.3-13.4 3.2-1.6 4.6-2.9 4.2-3.9-.3-.7-.5-1.8-.5-2.3 0-1.5-3.2-.9-9.5 1.7z"/><path d="M686.6 200.6c-8 .8-13.3 1.7-18.5 3.1l-5.5 1.3a106.7 106.7 0 0 0-28.2 12.5c-1.3 0-9.6 5.9-13 9.2-1.5 1.6-3.2 2.8-3.7 2.8s-1.3.9-1.6 2c-.3 1.1-1 2-1.4 2-.9 0-9 10.4-9 11.5 0 .3-1.2 2.2-2.4 4.3a53.2 53.2 0 0 0-5.7 13.4l-.5 1.8-1 4a102 102 0 0 0 1 35c-.7 0 5.8 12.6 8.6 16.6a104 104 0 0 0 39.8 30.6l20.1 9.5c12.6 6.1 24 14.2 24 17.2 0 .6.4 1.1 1 1.1 1.4 0 3.3 6 3.6 11.2.5 12.8-8 23-23.4 27.4-7.7 2.3-9 2.4-21.2 2.5-9 0-17.7-1-30.5-3.6-6.3-1.3-21-6.7-28.7-10.5-4.3-2.2-8.3-4-8.9-4-.5 0-1.2 1.3-1.5 3-.4 1.6-1 3-1.5 3s-.9.6-.9 1.4c0 1.1-3 8-10 23.1-6 12.9-7 15.9-5.8 16.7 4.4 2.9 21.2 10.3 30.1 13.3 16.4 5.3 30.4 7.5 48.7 7.5a166.4 166.4 0 0 0 48-5.7l2.5-.9a128 128 0 0 0 23-9.7 94 94 0 0 0 28.4-25 105 105 0 0 0 9.7-18.5l2.2-9.4c3-11.6 2.6-31.4-.5-38.8l-.8-2.6a68.8 68.8 0 0 0-17.6-25.7c-6.8-6-6-5.5-14.4-11a416.6 416.6 0 0 0-37-18.7l-7.6-4.2c-7.7-4.4-19.5-15.1-18.5-16.7.4-.5.2-1.1-.3-1.3-1.5-.5-2-8.4-.8-12.5 2.4-7.8 13-15.4 25.4-18.3a128 128 0 0 1 38 .4 151 151 0 0 1 24.1 7.2c9 3.7 9.3 3.8 9 2.6 0-.4.6-2.2 1.5-3.9 1-1.8 1.8-3.6 1.8-4 .2-1.2 5.7-15.8 7.8-20.8 6.4-14.8 7.6-19.5 5-19.5-.5 0-2.8-.9-5.2-1.9a160 160 0 0 0-19.5-6.6l-4.4-1a125.6 125.6 0 0 0-30.3-2.9c-9.3-.1-19.9.1-23.5.5z"/><path d="M410.4 204.3c-1.5.3-2 2-3.4 10.2l-2 10.3-2 10.5-1 5.7c-.9 3.5-1 4.1-1.9 9.5l-.7 4h32.2c27.6 0 32.1.2 31.7 1.4-1.5 7.2-3 14.4-4.3 21.6l-.9 4.6-1 6-1 5.4c-.3.8-.7 3.3-1 5.5l-1 5.7-1 4.5-2.6 13.8-2.4 13.5-1 5.5c-.4 1.6-.8 3.9-.9 5a9.8 9.8 0 0 1-1 3.4c-.4.7-.6 1.5-.3 1.7.3.3.1 2-.4 3.7a62 62 0 0 0-1.5 7.7c-.3 2.4-.7 4.7-.9 5a33 33 0 0 0-1 5 33 33 0 0 1-1 5c-.2.3-.6 3-1 6l-1 6.1c-.3.3-.7 2-1 3.7a1774.6 1774.6 0 0 1-12.7 69.2l-.3 2.5 28.7.1c31.6.1 29.3.6 30.4-6.5l2.4-13.1c1.1-5.2 2.1-10.9 2.4-12.6l2.1-11c1.2-5.4 2.2-10.9 3-16.4.9-3.3 1.5-6.6 2-10a137 137 0 0 0 2-11l1-5 3.1-16.5 3-15.5 1-5 2.4-13 2.5-13 1-5.5c.4-2.9.8-4.8 2-10.5l2-11c.6-4 1-5.6 2.3-12 1.1-4.4 2-9 2.6-13.5l1.1-6.3.7-4.2h57.8l.6-2.8a68 68 0 0 1 3-8c1.3-2.9 2.3-5.5 2.3-5.9 0-.3.3-.9.8-1.2.5-.3 2-2.2 3.3-4.1a54 54 0 0 1 13.7-13.7c-1.3-1.2 11.8-8.8 21.8-12.7 2.5-.9 3.3-1.5 2-1.6-5.4-.3-215.4-.5-216.7-.2zm-187.9.4-.4 2.8-2.4 13a763.3 763.3 0 0 0-6.6 34.1l-2 11.9-2 11-1.2 5.5c-.9 4-.8 3.4-1.8 9l-1 6-1 5.5c-.4 2.4-.8 4.7-1 5.1-.3.3-.6 2.2-.9 4.2-.2 2-.7 4.4-1 5.4l-1 5.3-2 11-1.2 6.5c-.3 2.2-.7 4.2-.9 4.5-.2.3-.6 2.6-1 5-.4 3-1 6-1.7 9-.5 2.5-1.2 5.8-1.4 7.5a2812.2 2812.2 0 0 0-6.8 36l-.9 5c-.4 1.6-1 4.7-1.3 6.9l-1 6c-1.2 5.4-2.5 12.4-3 15.6l-3 15.8-2.4 12.8c0 .2 39.4.4 87.5.4 69.1 0 87.7-.3 88-1.3.3-.6.7-2.8 1-4.7l.5-3.5.6-2.5 1.5-8 1.5-8 1.5-8 1.5-8c.2-1.4.7-3.1 1-3.8.4-1.4 5.7-1.3-65.8-1.3h-50.8l.5-3.7 1-6.2a3383 3383 0 0 0 8-43.8c.4-3 1.3-5.6 2-5.8.6-.3 24.3-.6 52.5-.7l51.4-.2.8-3 1.3-6.5c.7-5.7 1.8-11.3 3.1-16.8l1.6-8.5c.7-4 1.2-7.8 1.1-8.5 0-.6.2-1.2.6-1.2.4 0 .7-1 .6-2.2l-.2-2.2-51.4-.1-52-.3c-.4-.1 0-3.2.7-6.8.8-3.6 1.6-8 1.8-9.7l1.6-8.7 1.7-8.5 2-10c.7-3.9 1.5-7.5 1.5-8.1.1-.9 14.1-1.2 51.7-1.2l54.7-.1c3-.1 3.1-.2 4.4-7.4l1.6-8.2.8-4.9c.3-2.1.8-4.1 1-4.5.2-.3.7-2.8 1-5.5.4-2.7 1-5.2 1.1-5.5l1-4.5 1.1-6.2.6-2.4h-84c-46.4 0-84.2.1-84.3.2z"/></svg>
```

Reply:
@hkamran80 I used SVGOMG on this file. Are you using another SVG optimizer to generate this output?

Reply:
I removed the registered trademark symbol using Inkscape, adjusted the `viewBox` using it as well, then ran it through SVGOMG with all the features except "Remove xmlns" on, transform precision set to 0, number precision set to 1, and all global settings turned off (though I do use the "Show original" to make sure it looks mostly similar).